packages/swapper/src/swappers/RelaySwapper/utils/types.ts

```diff
@@ -94,16 +96,22 @@ export type QuoteDetails = {
   timeEstimate: number
 }
 
-export type RelayQuoteItemData = {
+export type RelayQuoteEvmItemData = {
```

Comment:
This becomes EVM specific and introducting `RelayQuoteUtxoItemData` and later Solana!